README.md

```diff
@@ -34,12 +34,27 @@ Advanced features of Taichi include [spatially sparse computing](https://docs.ta
 
 ## Installation [![Downloads](https://pepy.tech/badge/taichi)](https://pepy.tech/project/taichi)
 
+### Official releases
+
 ```bash
 python3 -m pip install taichi
 ```
 
 **Supported OS**: Windows, Linux, Mac OS X; **Python**: 3.6-3.9 (64-bit only); **Backends**: x64 CPUs, CUDA, Apple Metal, Vulkan, OpenGL Compute Shaders.
 
+### Nightly releases
+
+Nightly releases with the `master` branch of `taichi` are also available:
+
+```bash
+pip install -i https://test.pypi.org/simple/ taichi-nightly
```

Comment:
I remember this could cause a non-stopping installation?

Reply:
Hmmm it has been working for me. @qiao-bo Do you happen to remember what was the root cause last time it happened on your computer?

Reply:
It was random, did not happen again.